### PR TITLE
fix/snackbar

### PR DESCRIPTION
**Pull request by San88Francisco**
https://app.clickup.com/t/86958a0r8

### CODE REVIEW

File: src/components/Account-panel/Account-nav/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { CustomTabPanel } from '../../../utils/tab-panel/tab-panel'
+import {
+  stTab,
+  stTabsNav,
+  stContainerTabPanel,
+  stTabsBottomBox,
+  stWavePink
+} from '../style'
+
+type PropsType = {
```
Comment (pqoqubbw) [inline src/components/Account-panel/Account-nav/Tabs.tsx]:
```suggestion
type Props = {
```

File: src/components/Account-panel/Account-nav/Accordions.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { Box, Container, useMediaQuery, useTheme } from '@mui/material'
+import { Button } from '../../UI/Button'
+import { Typography } from '../../UI/Typography'
+
+import { useAuth } from '../../../hooks/use-auth'
+import { accordions } from '../../../constants/account/accordions'
+import { CustomAccordion } from '../../../utils/accordions-panel/accordions-panel'
+import { stAccordionBtn, stContainerTabPanel, stWavePink } from '../style'
+
+type PropsType = {
```
Comment (pqoqubbw) [inline src/components/Account-panel/Account-nav/Accordions.tsx]:
```suggestion
type Props = {
```

File: src/components/Account-panel/Account-nav/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+}) => {
+  const theme = useTheme()
+  const { user } = useAuth()
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'))
+
+  useEffect(() => {
+    setSearchParams({ tab: (activeIndex + 1).toString() })
+  }, [activeIndex])
+
+  const handleChange = (_event: SyntheticEvent, newValue: number) => {
```
Comment (pqoqubbw) [inline src/components/Account-panel/Account-nav/Tabs.tsx]: why do you pass `_event` if you do not use it?

File: src/constants/account/accordions.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { ChangePassword } from '../../components/Account-panel/Change-password/Change-password'
+import { ContactInfo } from '../../components/Account-panel/Contact-info/Contact-info'
+import { DeliveryAddress } from '../../components/Account-panel/Delivery-address/DeliveryAddress'
+import { OrderHistory } from '../../components/Account-panel/Order-history/Order-history'
+
+export const accordions = [
```
Comment (pqoqubbw) [inline src/constants/account/accordions.tsx]:
```suggestion
export const ACCORDIONS = [
```

File: src/utils/accordions-panel/accordions-panel.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography
+} from '@mui/material'
+import { FC, ReactNode } from 'react'
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import { a11yProps } from '../a11y-tabs/a11yTabs'
+
+type CustomAccordionProps = {
```
Comment (pqoqubbw) [inline src/utils/accordions-panel/accordions-panel.tsx]:
```suggestion
type Props = {
```

File: src/utils/tab-panel/tab-panel.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import { FC, ReactNode } from 'react'
+import { Box } from '@mui/material'
+
+type TabPanelProps = {
```
Comment (pqoqubbw) [inline src/utils/tab-panel/tab-panel.tsx]:
```suggestion
type Props = {
```

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/Account-panel/Account-nav/Tabs.tsx (hunk 1 above)
Comment (San88Francisco) [reply src/components/Account-panel/Account-nav/Tabs.tsx]: done

File: src/components/Account-panel/Account-nav/Accordions.tsx (hunk 2 above)
Comment (San88Francisco) [reply src/components/Account-panel/Account-nav/Accordions.tsx]: done

File: src/components/Account-panel/Account-nav/Tabs.tsx (hunk 3 above)
Comment (San88Francisco) [reply src/components/Account-panel/Account-nav/Tabs.tsx]: corrected

File: src/utils/accordions-panel/accordions-panel.tsx (hunk 5 above)
Comment (San88Francisco) [reply src/utils/accordions-panel/accordions-panel.tsx]: done

File: src/constants/account/accordions.tsx (hunk 4 above)
Comment (San88Francisco) [reply src/constants/account/accordions.tsx]: done

File: src/utils/tab-panel/tab-panel.tsx (hunk 6 above)
Comment (San88Francisco) [reply src/utils/tab-panel/tab-panel.tsx]: done

File: src/components/modal-portal/ModalPortal.tsx [hunk 7]
```diff
@@ -17,36 +13,13 @@ type Props = {
...
-    if (searchToken) {
-      const searchKey = Object.keys(searchToken)[0]
-      searchParams.delete(searchKey)
-      navigate({ search: searchParams.toString() })
-      setIsModalOpen(false)
-      return
-    }
-    setIsModalOpen(false)
-  }
-
```
Comment (pqoqubbw) [inline src/components/modal-portal/ModalPortal.tsx]: ?

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/modal-portal/ModalPortal.tsx (hunk 7 above)
Comment (San88Francisco) [reply src/components/modal-portal/ModalPortal.tsx]: ops....

Comment (San88Francisco) [reply src/components/modal-portal/ModalPortal.tsx]: corected

**Pull request merged**